include/private/gcconfig.h

```diff
@@ -748,6 +752,7 @@ EXTERN_C_BEGIN
                     /*             HEXAGON    ==> Qualcomm Hexagon      */
                     /*             TILEPRO    ==> Tilera TILEPro        */
                     /*             TILEGX     ==> Tilera TILE-Gx        */
+                    /*             LOONGARCH  ==> Loongson loongarch64  */
```

Comment:
别的地方已经改了没有体现位数了，这里又 loongarch64 不一致。

看其他写得比较好的条目，显然这里要表明公司名、架构名、支持的操作系统、位数等情况。建议缝合一下。

```suggestion
                    /*             LOONGARCH  ==> Loongson LoongArch    */
                    /*                  running LINUX                   */
                    /*                  Handles 32 and 64-bit variants. */
```

Reply:
也考虑到了， 可是现在没有32位，要写上吗？

Reply:
ok, 已更正